### PR TITLE
Replace manual Kubernetes manifests with Helm chart

### PR DESCRIPTION
**Pull request by djones6**
- Replaces the manual Strimzi install with a helm install
- Replaces the deploy.yml + service.yml for each service with a Helm chart
- Adds the app's topic configuration to the Helm chart (Kafka is created in a separate step, because Helm cannot be told to wait for the output of the Strimzi operator)
- Adds an `install.sh` and `uninstall.sh` that installs / uninstalls all of the above.
- Updates Travis to test the helm deployment.

### CODE REVIEW

File: uninstall.sh [hunk 1]
```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
```
Comment (Cottonglow) [inline uninstall.sh:1]: Should we include an uninstall.bat script for the windows users as well?

File: README.md [hunk 2]
```diff
@@ -215,3 +215,38 @@ The dashboard shows that the load is dispatched among the baristas.
     ```bash
     ./delete-kubernetes.sh
     ```
+
+# Helm (v3)
+
+Note: current problems: Although Strimzi's kafkatopics resource creates the 'orders' topic with multiple partitions, the partition count then seems to get reset to 1, which means Keda can't scale the barista. As a workaround, changing the partition count and then upgrading the chart seems to resolve this (usually).
```
Comment (Cottonglow) [inline README.md:221]: I assume the partition count is changed in the file? If so, there is a script you can run locally as well, seeing as you have nodeport on kafka:
`./kafka-topics.bat --bootstrap-server localhost:32100 --alter --topic orders--partitions 5`

This way you don't need to upgrade the chart after. I'm not sure how useful this is though, so up to you!

**Review by Cottonglow — APPROVED** [review]
Looks good :)

**Review by matthewpwilson — APPROVED** [review]
Looks fine as far as I can tell!

File: README.md (hunk 2 above)
Comment (djones6) [reply README.md:221]: Oh good point, though I can probably get rid of this statement.  I think I have resolved it by creating the Kafka cluster as an earlier step.  If Kafka and Strimzi are already running, the topics seem to be created with the right parameters reliably.

**Pull request merged**